RELEASE_NOTES.md

```diff
@@ -1,30 +1,21 @@
 # Edge Image Builder Releases
 
-# Next
```

Comment:
We need to keep this section.

Reply:
I'll add them back in after the 1.1 branch. Felt cleaner to have this look like a finished release.